opennms-config-model/src/main/java/org/opennms/netmgt/config/discovery/Definition.java

```diff
@@ -204,6 +211,25 @@ public boolean removeIncludeUrl(final IncludeUrl includeUrl) {
         return includeUrls.remove(includeUrl);
     }
 
+    public List<ExcludeUrl> getExcludeUrls() {
+        return excludeUrls;
+    }
+
+    public void setExcludeUrls(final List<ExcludeUrl> excludeUrls) {
+        if (excludeUrls == this.excludeUrls) return;
```

Comment:
Removed the changes not required for discovery
Applied changes sugested to use equal rather than == when comapring ExcludeUrl Instances
Made Url field mandatory rather than Optional in the ExcludeUrl class (and the related changes)